src/services/deleteItemFromFirebase.tsx

```diff
@@ -1,8 +1,10 @@
 import { db } from '../firebase';
-import { doc, deleteDoc } from 'firebase/firestore';
+import { doc, updateDoc } from 'firebase/firestore';
 import { getAuth } from 'firebase/auth';
 
-export const deleteItemFromFirebase = async (category, itemId) => {
+import { arrayRemove } from 'firebase/firestore';
+
+export const deleteItemFromFirebase = async (category: string, item: Item) => {
```

Comment:
category to może być unia typów np `type CategoryTypes = 'spells' | 'weapons' | 'costam'` tak aby inni developerzy nie używali dziwnych stringów, które nie są obsługiwane

Reply:
Do tego jeszcze można dorobić enuma 

enum Categories = {
 SPELLS = "spells",
 ... reszta
}

i bedzie typ `type CategoryTypes = Categories.SPELLS | Categories.WEAPONS | ...reszta`